src/meta-srv/src/service/heartbeat.rs

```diff
@@ -100,7 +100,7 @@ impl heartbeat_server::Heartbeat for MetaSrv {
                 pusher_key.as_ref().unwrap_or(&"unknow".to_string())
             );
             if let Some(key) = pusher_key {
-                let _ = handler_group.unregister(&key);
+                let _ = handler_group.unregister(&key).await;
```

Comment:
What did clippy say? Looks like `.await` is needed.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#let_underscore_future